reporting/sndbox.py

```diff
@@ -157,20 +159,26 @@ def run(self, results):
                 debug,
                 process_error  # pass the error to the user
             )
+            return
+
 
-        elif debug['errors'] or has_no_behavior:
+        if has_errors_or_no_behavior and not is_office_package:
             # don't remove from the queue, retries might be
             # helpful here since this might be an issue with this host only.
             logger.warning("No behavior was found")
             results["reporting_status"] = "nobehavior"
+            return
 
-        elif self.has_crash_process(results["behavior"]["processes"]) and not custom["soon_to_retire"]:
+        if has_crash_process_and_should_retry and not is_office_package:
             # in case we have a crash process and this sample is scheduled to have more retries, we should
             # run it again, this handles case where VMs sometimes perform poorly under heavy load
             logger.warning("Detected crash process")
             results["reporting_status"] = "crash"
+            return
 
-        else:
-            self.send_success_notification(results["s3"], sample)
-            self._sqs.delete_message(QueueUrl=custom['source_queue'], ReceiptHandle=custom['receipt_handle'])
-            results["reporting_status"] = "completed"
+        if (has_errors_or_no_behavior or has_crash_process_and_should_retry) and is_office_package:
+            logger.warning("ignored a detected crash, package is office")
```

Comment:
No, this is not the same, you shouldn't handle no behavior the same way you handle a crash.

you cannot continue if you have no behavior